### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RickLiu0121**
I just added the build instruction. The leads plz add more details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
 ## Table of Contents
-- [SYDE2025-1A-Class-Profile-](#syde2025-1a-class-profile-)
+- [SYDE2025-2A-Class-Profile](#syde2025-1a-class-profile-)
   * [About The Project](#about-the-project)
-    + [Purpose](#purpose)
-    + [Languages, Tools and Libraries](#languages-tools-and-libraries)
+  * [Languages, Tools and Libraries](#languages-tools-and-libraries)
+  * [Build instruction](#build-instruction)
   * [License](#license)
   * [Contact](#contact)
```
Comment (rayasfour) [inline README.md:10]: What's the point of this? 

Add "Contact" and "License" sections or remove L9-10.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
 
 ## About The Project
-The SYDE class profile, was created to give an insight to the SYDE 2025 cohort's 1A term, similar to reports created
-by previous cohorts. We wanted to go a step above and create a website that would document not just our 1A term, but future terms as well.
+The SYDE class profile, was created to give an insight to the SYDE 2025 cohort's 2A term, similar to reports created
+by previous cohorts. We wanted to go a step above and create a website that would document not just our 2A term, but future terms as well.
```
Comment (rayasfour) [inline README.md:14]: I would suggest adding a sentence or two on the purpose of this project.

Something along the lines of "We wanted to discover and document the impact of COVID-19 on the students of our cohort"

File: README.md [hunk 3]
```diff
@@ -1,19 +1,28 @@
...
 
-### Languages, Tools and Libraries
+## Build instruction 
+- On develop branch, clone the repo (git clone [URL])
+- cd into folder
+- Run the following commands
+  - ```yarn install```
```
Comment (rayasfour) [inline README.md:20]: Merely a personal preference but I like npm install, u seem to be using npm commands later anyway

Comment (RickLiu0121) [reply README.md:20]: npm install didn't really work for me. It was showing dependency errors. I used yarn and it worked fine.

**Pull request merged**